modules/BaadalMailer.py

```diff
@@ -29,6 +29,12 @@ class ApprovalReminder(object):
                 "'{0[requestType]}' request on {0[requestTime]}. It is "\
                 "waiting for your approval."
 
+        class IPRequest(object):
```

Comment:
A notification email after the public IP has been attached would be more helpful.